### PR TITLE
fix(rest): componentType field  will show when allDetails true in rel…

### PR DESCRIPTION
**Pull request by nikkuma7**
…ease

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? #1878 
> * Did you add or update any new dependencies that are required for your change? No

Issue: 
Added field componentType when allDetails=true in release. 

### How To Test?
Please hit the below url and check whether componentType field is present or in release block. 
/resource/api/releases?allDetails=true
image

> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/Sw360ReleaseService.java [hunk 1]
```diff
@@ -131,6 +131,7 @@ public void setComponentDependentFieldsInRelease(Release releaseById, User sw360
             throw new HttpMessageNotReadableException("No Component found with Id - " + componentId);
         }
         releaseById.setComponentType(componentById.getComponentType());
+        return releaseById;
```
Comment (rudra-superrr) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/Sw360ReleaseService.java:135]: remove this extra line

Comment (rudra-superrr) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/Sw360ReleaseService.java:135]: remove this extra line.

**Comment by rudra-superrr** [comment]
Testing was successful.

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/Sw360ReleaseService.java (hunk 1 above)
Comment (nikkuma7) [reply rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/Sw360ReleaseService.java]: done

**Pull request merged**